flask_appbuilder/validators.py

```diff
@@ -1,26 +1,46 @@
-from wtforms import ValidationError
+import re
+from typing import Optional
 
+from flask import current_app
+from flask_appbuilder.exceptions import PasswordComplexityValidationError
+from flask_appbuilder.models.base import BaseInterface
+from flask_babel import gettext
+from wtforms import Field, Form, ValidationError
 
-class Unique(object):
+password_complexity_regex = re.compile(
+    r"""(
+    ^(?=.*[A-Z].*[A-Z])                # at least two capital letters
+    (?=.*[!@#$&*])                     # at least one of these special characters
```

Comment:
To not "force" users to use a highly restricted set of special characters, would it be better to do a negative match for non-special characters here? Something like 
```regex
(?!([a-zA-Z0-9])).
```

Reply:
good point